### PR TITLE
feat(core): add translation key for payment

### PR DESCRIPTION
**Pull request by JimmFly**

### CODE REVIEW

File: packages/frontend/core/src/components/affine/setting-modal/general-setting/billing/index.tsx [hunk 1]
```diff
@@ -77,61 +81,68 @@ const SubscriptionSettings = () => {
...
-                </a>
-                .
-              </p>
+              <Trans
+                i18nKey="com.affine.billing.current-plan.description"
+                values={{
+                  planName: planText,
+                }}
+              >
+                You are current on the <a>planName</a> plan.
```
Comment (forehalo) [inline packages/frontend/core/src/components/affine/setting-modal/general-setting/billing/index.tsx:101]: `{planName}`?

Comment (JimmFly) [reply packages/frontend/core/src/components/affine/setting-modal/general-setting/billing/index.tsx:101]: It has two uses:
```
  <Trans
       i18nKey="com.affine.billing.current-plan.description"
       values={{
           planName: planText,
       }}
   >
       You are current on the <a>planName</a> plan.
   </Trans>
 ```
 or
 ```
   <Trans  i18nKey="com.affine.billing.current-plan.description" >
       You are current on the <a>{planText}</a> plan.
   </Trans>
```

File: packages/frontend/i18n/src/resources/en.json [hunk 2]
```diff
@@ -637,5 +639,33 @@
   "com.affine.auth.sign-out.confirm-modal.title": "Sign out?",
   "com.affine.auth.sign-out.confirm-modal.description": "After signing out, the Cloud Workspaces associated with this account will be removed from the current device, and signing in again will add them back.",
   "com.affine.auth.sign-out.confirm-modal.cancel": "Cancel",
-  "com.affine.auth.sign-out.confirm-modal.confirm": "Sign Out"
+  "com.affine.auth.sign-out.confirm-modal.confirm": "Sign Out",
+  "com.affine.storage.maximum-tips": "You have reached the maximum capacity limit for your current account",
+  "com.affine.price-plan.free": "Free",
```
Comment (joooye34) [inline packages/frontend/i18n/src/resources/en.json:644]: Those type seems no need to translate

File: packages/frontend/i18n/src/resources/en.json [hunk 3]
```diff
@@ -637,5 +639,33 @@
...
+  "com.affine.price-plan.pro": "Pro",
+  "com.affine.price-plan.team": "Team",
+  "com.affine.price-plan.enterprise": "Enterprise",
+  "com.affine.price-plan.tag-tooltips": "See all plans",
```
Comment (joooye34) [inline packages/frontend/i18n/src/resources/en.json:648]: use `com.affine.payment.xxxx`

File: packages/frontend/component/src/components/setting-components/storage-progess.tsx [hunk 4]
```diff
@@ -30,11 +30,11 @@ export const StorageProgress = ({
   const max = useMemo(() => bytes.format(upperLimit), [upperLimit]);
 
   const buttonType = useMemo(() => {
-    if (plan === 'Free') {
+    if (plan === t['com.affine.price-plan.free']()) {
```
Comment (joooye34) [inline packages/frontend/component/src/components/setting-components/storage-progess.tsx:33]: We need to define enum, maybe isomorphic constants

**Review by CatsJuice — APPROVED** [review]

**Review by joooye34 — APPROVED** [review]

**Pull request merged**